docs/README.md

```diff
@@ -52,7 +52,9 @@ To specify different menu text from the page title, use the front matter paramet
 
 ### Add images
 
-Images are currently hosted in the grafana/website repo.
+Any new images should be uploaded to Grafana's [media manager](https://admin.grafana.com/upload/media/). At the moment, legacy images are still hosted in the `grafana/website` repo. 
```

Comment:
I would just remove this line in favor of the one below that links to the guidelines.

Reply:
The images aren't "legacy" - they can still be selected and used in new content via the uploader.